config.toml

```diff
@@ -6,42 +6,28 @@
 # Main Config
 [main]
 data-folder = "~/.spacemesh-data"
-log-encoder = "console"
 metrics = false
 metrics-port = 2020
 oracle_server = "http://localhost:3030"
 oracle_server_worldid = 0
 genesis-time = "2019-02-13T17:02:00+00:00"
 layer-duration-sec = "5"
-block-cache-size = "20"
-hdist = "5"
-coinbase = "0x1234"
```

Comment:
a lot of entries here were deleted, are they really not configurable any more?

Reply:
i don't know why we keep this config in repo

Reply:
While I am at it, I can remove it completely 🤷 .